src/content/docs/style-guide/writing-docs/writer-workflow/tech-writer-workflow.mdx

```diff
@@ -172,4 +172,20 @@ To create a branch on the docs-website repo:
 3. Click on **New Branch**
 4. You will be prompted to name your new branch. Descriptive names are best. It's a great way to quickly clue people in to what your work is all about. For example, if you are working on What’s New pages, you might name the branch **Whats-new-updates**.
 
-When you create a new branch, don't forget to add the Jira ticket's key (DOC-1234) to the branch name and the PR title. 
\ No newline at end of file
+When you create a new branch, don't forget to add the Jira ticket's key (DOC-1234) to the branch name and the PR title. 
+
+## Reviewing quickstart descriptions
+
+External vendors write quickstart descriptions. When one of these are ready for a review, they open an issue in our repo with a link out to the draft. These are written with SEO in mind, so they will appear differnt than most of our other docs and will not fully follow our style guide. 
+
+To close these issues:
+
+1. Follow the link to the draft.
+2. Review the draft with the tips below. 
```

Comment:
@bradleycamacho, how about this: Review the draft using the tips below.

Reply:
@bradleycamacho, it looks like the drafts are in Google docs, so you might be more explicit about turning on suggestions and adding comments in the doc instead of just changing the doc.